.github/workflows/conformance-kind-proxy-daemonset.yaml

```diff
@@ -65,18 +65,18 @@ jobs:
             --helm-set=clustermesh.apiserver.image.repository=quay.io/${{ env.QUAY_ORGANIZATION_DEV }}/clustermesh-apiserver-ci \
             --helm-set=clustermesh.apiserver.image.tag=${SHA} \
             --helm-set=clustermesh.apiserver.image.useDigest=false \
-            --helm-set hubble.relay.enabled=true \
+            --helm-set=hubble.relay.enabled=true \
             --helm-set=hubble.relay.image.repository=quay.io/${{ env.QUAY_ORGANIZATION_DEV }}/hubble-relay-ci \
             --helm-set=hubble.relay.image.tag=${SHA} \
-            --helm-set cni.chainingMode=portmap \
-            --helm-set loadBalancer.l7.backend=envoy \
-            --helm-set tls.secretsBackend=k8s \
-            --helm-set envoy.enabled=true \
-            --helm-set bpf.monitorAggregation="none" \
+            --helm-set=cni.chainingMode=portmap \
+            --helm-set-string=kubeProxyReplacement=strict \
+            --helm-set=loadBalancer.l7.backend=envoy \
+            --helm-set=tls.secretsBackend=k8s \
+            --helm-set=envoy.enabled=true \
+            --helm-set=bpf.monitorAggregation="none" \
             --wait=false \
             --version="
-          CONNECTIVITY_TEST_DEFAULTS="--flow-validation=disabled --hubble=false --collect-sysdump-on-failure \
-            --external-target bing.com --external-cidr 8.0.0.0/8 --external-ip 8.8.8.8 --external-other-ip 8.8.4.4"
```

Comment:
This change may not help because the default values are 1.1.1.1 and 1.0.0.1, which are also flaky.
The last time I tested connectivity from Azure, which I assume GitHub runners work in, Google IPs gave much better results in terms of connectivity time.

Why do you think this will help?

Reply:
@brlbil it looks like e.g. `--external-ip 8.8.8.8` is only used for GKE related jobs on `main`.

It's just that this job with these settings started to become very flaky in the past days (spotted on `main` and my own PRs) - whereas other jobs executing the connectivity jobs seemed to run fine. 

And as mentioned above - on this PR i have 4 successful runs in a row from the beginning.

https://github.com/cilium/cilium/actions/workflows/conformance-kind-proxy-daemonset.yaml